src/main/webapp/resources/js/components/DefaultErrorBoundary.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Alert } from "antd";
+
+interface DefaultErrorBoundaryProps {
+  message?: string;
+  description?: string;
+}
+
+/**
+ * Default error boundary for react-router-dom
+ * @param message "message" attribute for Alert component
+ * @param description "description" attribute for Alert component
+ * @constructor
+ */
+function DefaultErrorBoundary({
```

Comment:
Does this just happen is a user is attempting to access an invalid route?

Reply:
No, this happens if there is an error getting data in the loader component or there is any other error rendering the component.  Right now I just created this default, but we can definitely update in the future to use more specific ones.  It prevents the error from crashing the whole page, just the component under the route fails to load.